templates/debug/log_context.hbs

```diff
@@ -0,0 +1,17 @@
+<script type="text/javascript">
+   (function(){
+       {{#if debug-context}}
+          var plaintext = "{{debug-context}}";
+       {{else}}
+          var plaintext = "{{this}}";
+       {{/if}}
+       try{
+           var pre = document.createElement("pre");
```

Comment:
Do we actually need to create an element here? What happens if we just `JSON.parse` the plaintext?